### ISSUE TITLE
Player Error

### ISSUE THREAD

**Issue by atleticano-snp** [Saibot393/Rideable#45]
Hi,
 
My players login are getting this error, everytime a token that they do not own moves in the canvas. This error started yesterday when I was testing some NPC.

![image](https://github.com/Saibot393/Rideable/assets/68756964/8a60f8b8-cbac-4dcf-b42f-109b8a69cbc1)

Firstly I deactived some modules to find out that Rideable was the culprit. 
The error persisted even when it was the only module in the world.
When I created a fresh new world and only enabled rideable, the error persisted.

Rideable v2.14.3
System:  Pathfinder 2e V. 5.7.4

**Comment by atleticano-snp** [comment]
Sure, sorry. Cleaned the console first, than as GM moved the token and printed the console from the player view.

![image](https://github.com/Saibot393/Rideable/assets/68756964/b6b46297-6f99-4c11-aa76-7a7a5ae18135)

**Comment by Saibot393** [comment]
The new v2.14.4 should solve the issue

**Comment by Tetrachloroethene** [comment]
I'm also having this issue (as a player, Foundry v11.313, Game System DnD5e 2.3.1). Had installed module version 2.14.3 before the session, disabled this module and errors stopped. Upgraded this module to 2.14.5 after finding this issue, errors persisted.

Didn't manage to grab logs until we disabled the module to play the session, but can spin up another world later to experiment and pull logs from if that would be useful.

**Comment by Saibot393** [comment]
I did a few tests and can no longer replicate it. A screenshot of the error log would be apreciatet

**Comment by Saibot393** [comment]
I have fixed an error with v2.15.11 that may have been connected to this issue

**Comment by Tetrachloroethene** [comment]
Sorry, I was sure I'd replied after testing but I must have not actually submitted it - I did some testing on some fresh instances of Foundry and was unable to replicate the problem I had (installing 2.14.3, upgrading to 2.14.5). I then redid the module upgrade to 2.14.5 on the main instance, and it worked fine, no errors.

I think it was just something weird that happened with my specific instance not performing the upgrade correctly- other than that I've not seen the issue again on any version 2.14.5+ (didn't try 2.14.4, so likely was fixed when you suggested).
